onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -129,6 +129,8 @@ namespace Dml
         }
     }
 
+#define D3D_FEATURE_LEVEL_1_0_CORE_PRIVATE ((D3D_FEATURE_LEVEL)0x1000)
```

Comment:
this is a temporray thing to get around the fact that we don't have the 19h1 SDK on our build machines.  let's put a nice comment  here in the code saying why we did this, and link to the tracking bug# that will fix it correclty long term.   we should fix this before we hit ORT 1.2